cert_manager/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG - cert_manager
 
+## 3.0.0 / 2022-09-16
+
+* [Changed] Migrate to OpenMetrics V2. See [#12344](https://github.com/DataDog/integrations-core/pull/12344).
+* [Added] Move cert_manager to core. See [#12344](https://github.com/DataDog/integrations-core/pull/12344).
```

Comment:
This changelog entry's link is the same as the one above?

Reply:
We did both things in the same PR. We migrated it at the same time. Wanted to make it clear that the breaking change was related to OpenMetrics V2 and not to the fact that this integration has been moved to -core

Reply:
Ahh gotcha. Thanks!